### PR TITLE
[ko] Machine translation of missing keys

### PR DESCRIPTION
**Pull request by andrey-helldar**

### CODE REVIEW

**Comment by andrey-helldar** [comment]
@SilNex, hey! We used machine translation of the text. Can you check if the translation is correct this PR? Thank you.

File: locales/ko/ko.json [hunk 1]
```diff
@@ -333,7 +333,7 @@
     "Nepal": "네팔",
     "Netherlands": "네덜란드",
     "Nevermind": "괜찮습니다",
-    "New": "New",
```
Comment (SilNex) [inline locales/ko/ko.json:336]: better to it as "New".

**Review by SilNex — COMMENTED** [review]
that's ok
but better to leave it as "New".

**Comment by caouecs** [comment]
Ok thank you, I do updates ;)

**Comment by caouecs** [comment]
done

**Pull request merged**